tabs/src/components/App.jsx

```diff
@@ -27,7 +26,6 @@ export default function App() {
           <>
```

Comment:
why there is an empty tag?

Reply:
It's a react syntactic sugar which is short for Switch